composer.json

```diff
@@ -26,5 +27,8 @@
         "psr-4": {
             "Aura\\Marshal\\": "tests/"
         }
+    },
+    "scripts": {
+        "test": "phpunit"
```

Comment:
It looks phpunit is global on your end. But if you really need this to work, you need to use "./vendor/bin/phpunit"